test/unit/server_test.js

```diff
@@ -1117,5 +1117,556 @@ describe("server.js tests", function () {
           })
       });
     });
+
+
+    describe("AssetsCallBuilder", function() {
+      let assetsResponse = {
+        "_links": {
+          "self": {
+            "href": "http://horizon-testnet.stellar.org/assets?order=asc\u0026limit=10\u0026cursor="
```

Comment:
If we shape these objects according to the API, we might as well use the tested hostname which has a port 1337, `https://horizon-live.stellar.org:1337`, for all links. This currently doesn't break any tests, but should for more complete coverage.